### DISCUSSION TITLE
Extract from a PDF URL?

### DISCUSSION THREAD

**Discussion by Olney1** [jsvine/pdfplumber#485] [Q&A] [answered]
Hi, is it possible to extract text from a PDF file by opening a URL to the document or does the document need to be on a local drive?

**Comment by samkit-jain** [comment, marked as answer]
Hi @SentBackInTime There's no direct support for loading PDFs directly from a URL. You can however download the file from the URL and then pass it to pdfplumber. An example can be found at https://github.com/jsvine/pdfplumber/issues/132#issuecomment-519426102.
